.github/workflows/ci.yml

```diff
@@ -156,8 +158,7 @@ jobs:
         working-directory: target/snitch_cluster
         run: |-
           ./run.py --simulator verilator \
-          sw/runtime.yaml sw/standard-fp.yaml \
-          sw/snax-streamer-gemm-conv-run.yaml -j
+          sw/runtime.yaml sw/snax-streamer-gemm-conv-run.yaml -j
```

Comment:
Maybe return the `sw/standard-fp.yaml` since it now passes with our tests right? 😄

Reply:
Only the original snitch cluster supports FP, and all scalar tests are not removed.

Reply:
Ah i didn't see it was for Gemm LOL okay okay